rasa_sdk/endpoint.py

```diff
@@ -127,13 +127,14 @@ async def actions(_) -> HTTPResponse:
 
 def run(
     action_package_name: Union[Text, types.ModuleType],
-    port: Union[Text, int] = DEFAULT_SERVER_PORT,
+    port: int = DEFAULT_SERVER_PORT,
```

Comment:
if we were allowing `Text` here (likely from an env var?) should we cast the value below with `int(port)` instead of only changing the type?

Reply:
I had walked back the code inside rasa sdk and saw that it is only called with the args.port which is forced to be int. Also took another look rasa oss and rasa x also don't call this. Using `int(port)` might be safer for user code though

Reply:
ok sounds good then, if the user does not call this directly, I don't think it's necessary to typecast it

Reply:
At least we don't seem to call it directly in rasa oss and x. But users could be doing that, that's what I meant to say. Usually we would probably want a deprecation warning here - but given that it's such a small change we might just do it and get rid of a bit of clutter?

Reply:
no I think it's fine. If it's not in our guidelines to have user call this method directly, I think it's fine